db/migrate/20240117180824_latin1_to_utf8.rb

```diff
@@ -0,0 +1,15 @@
+class Latin1ToUtf8 < ActiveRecord::Migration[7.1]
+  def up
+    Article.connection.
+      execute("ALTER TABLE articles CONVERT TO CHARACTER SET utf8mb4;")
```

Comment:
Comment: MySQL recommends utf8mb4.  See https://dev.mysql.com/blog-archive/mysql-8-0-when-to-use-utf8mb3-over-utf8mb4/
Maybe I should make it the db default.

Reply:
Yes, I vote to make it the db default.

Reply:
I'll do that in another PR. 
It's probably safe, but I'd like to think about its consequences.